root/edit/details/AddReleaseGroup.js

```diff
@@ -64,7 +64,7 @@ const AddReleaseGroup = ({allowNew, edit}: Props): React$MixedElement => {
               <th>
                 {type.historic
                   ? addColonText(l('Type'))
-                  : l('Primary Type:')}
+                  : l('Primary type:')}
```

Comment:
As changing the capitalization will mark the message as fuzzy, we should probably drop the colon at the same time:
```suggestion
                  : l(addColonText('Primary type'))}
```

Reply:
Ok, did the same to the ones being changed in release and release group pages then.